openlibrary/templates/type/edition/view.html

```diff
@@ -344,6 +347,15 @@ <h3 class="list-header collapse">$_("ID Numbers")</h3>
 
         <div class="editionTools">
             $:macros.databarWork(page, editions_page=True)
+
+            $if page.donor and page.donor_msg:
+              <div class="Tools read-options">
```

Comment:
That's not what these classes are for; if you're going to use them for something else, rename them to something more generic. Otherwise, just use new classes :)

Reply:
Done